github/teams_discussions.go

```diff
@@ -44,12 +44,12 @@ type DiscussionListOptions struct {
 	Direction string `url:"direction,omitempty"`
```

Comment:
From the documentation, it looks like `DiscussionListOptions` also needs to embed a `ListOptions` struct to support pagination. Please add this.

Reply:
That's correct, I've updated it.